build-kong.sh

```diff
@@ -10,9 +10,9 @@ rocks_trees = {
 }
 " > $ROCKS_CONFIG
 
+[ -d "/usr/local/share/lua/" ] && cp -R /usr/local/share/lua/ /tmp/build/usr/local/share/ 
```

Comment:
Could be wrong, but I _think_ this will cause errexit (`set -e`) to bail out and fail if `/usr/local/share/lua` does not exist. Is that the intended behavior? If not I think this must be wrapped in an `if`

```suggestion
if [[ -d /usr/local/share/lua ]]; then
  cp -R /usr/local/share/lua/ /tmp/build/usr/local/share/ 
fi
```